tests/test_uharfbuzz.py

```diff
@@ -73,7 +73,9 @@ def test_guess_set_segment_properties(self):
 
         assert buf.direction == "rtl"
         assert buf.script == "Hebr"
-        assert buf.language == "en-us"
+        # the guessed language seems to be locale specific
+        # assert buf.language == "en-us"
```

Comment:
Yes, HarfBuzz just the current locale for guessing the language (that is why `guess_set_segment_properties` should not be used in production code in general).